cubedash/summary/_model.py

```diff
@@ -100,17 +99,33 @@ def add_periods(
                 else None
             )
         except ValueError:
-            _LOG.warn("summary.footprint.union", exc_info=True)
             # Attempt 2 at union: Exaggerate the overlap *slightly* to
             # avoid non-noded intersection.
             # TODO: does shapely have a snap-to-grid?
-            geometry_union = (
-                shapely.ops.unary_union(
-                    [p.footprint_geometry.buffer(0.001) for p in with_valid_geometries]
+            # mpolygon = MultiPolygon([polygon for polygon in [p.footprint_geometry for p in with_valid_geometries]])
+            try:
+                _LOG.warn("summary.footprint.union", exc_info=True)
```

Comment:
Now that we're splitting these apart it might be worth cleaning up my log names

```suggestion
                _LOG.warn("summary.footprint.invalid_union", exc_info=True)
```